TitleRoulette.sln

```diff
@@ -3,18 +3,18 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.4.33122.133
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TitleRoulette", "TitleRoulette.csproj", "{EC98CE71-953A-442B-9943-AB6EA57167D3}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TitleRoulette", "TitleRoulette.csproj", "{EC98CE71-953A-442B-9943-AB6EA57167D3}"
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{EC98CE71-953A-442B-9943-AB6EA57167D3}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{EC98CE71-953A-442B-9943-AB6EA57167D3}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{EC98CE71-953A-442B-9943-AB6EA57167D3}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{EC98CE71-953A-442B-9943-AB6EA57167D3}.Release|Any CPU.Build.0 = Release|Any CPU
+		{EC98CE71-953A-442B-9943-AB6EA57167D3}.Debug|Any CPU.ActiveCfg = Debug|x64
+		{EC98CE71-953A-442B-9943-AB6EA57167D3}.Debug|Any CPU.Build.0 = Debug|x64
+		{EC98CE71-953A-442B-9943-AB6EA57167D3}.Release|Any CPU.ActiveCfg = ExportRelease|x64
```

Comment:
I don't understand where you take `ExportRelease` from, Visual Studio doesn't even list that as one of the available configurations.

It's a good idea though to fix the configurations, I've changed that in https://github.com/mabako/TitleRoulette/commit/14ced583cd50c863f13284a88b02d7d13a8eb2f5

Reply:
That was because my VS didn't accept the existing config and the Configuration Manager from VS fixed it like that. Can be ignored if it works for you without.